packages/sanity/src/core/form/field/useFieldActions.ts

```diff
@@ -0,0 +1,28 @@
+import {useContext, useMemo} from 'react'
+import {DocumentFieldActionNode} from '../../config'
+import {FieldContext} from './FieldContext'
+
+/** @internal */
+export function useFieldActions(): DocumentFieldActionNode[] {
+  const {actions} = useContext(FieldContext)
+
+  return useMemo(() => filterActions(actions), [actions])
+}
+
+function filterActions(actions: DocumentFieldActionNode[]): DocumentFieldActionNode[] {
+  return actions
+    .filter((node) => {
+      if ('hidden' in node) return node.hidden !== true
+      return true
```

Comment:
<img width="940" alt="image" src="https://github.com/sanity-io/sanity/assets/406933/e93dd098-7173-44f6-8227-56fc28a72ffa">

This won't work, since the `type: "divider"` node doesn't have a `hidden` property. Also `undefined` should have the same meaning as `false`.

Reply:
ah, typescript :)